### DISCUSSION TITLE
Citing qmsolve in a physics paper

### DISCUSSION THREAD

**Discussion by MarkDPhysics** [quantum-visualizations/qmsolve#29] [Q&A]
I'm writing a physics paper and I think I'd like to cite qmsolve.  Can you suggest a citation that I can put into a bibliography or how I should cite it?

**Comment by rafael-fuente** [comment]
> I'm writing a physics paper and I think I'd like to cite qmsolve. Can you suggest a citation that I can put into a bibliography or how I should cite it?


Hi MarkDPhysics, we are discussing it. Some people cite just the repository link, but we will add a unique DOI. We'll let you know when it's done :)

By the way, what are you working on?

**Comment by MarkDPhysics** [comment]
I'm working on simulating Bohmian trajectories and stochastic mechanics
trajectories for single-particle time-dependent Shroedinger equations. For
example the two-slit diffraction problem. This is for a book chapter that
I've been asked to write on the Heisenberg algebra and stochastic models of
quantum mechanics on how they they are related. Actually the first version
of a preprint is already on the CERN repository called Zenodo if you are
interested:
Deriving Heisenberg's operator algebra from a classical model of stochastic
mechanics (zenodo.org) <https://zenodo.org/records/10277839>
I want to spruce this up with some visual examples using qmsolve.
Mark

**Comment by rafael-fuente** [comment]
We added an unique DOI for the project and a citation file https://github.com/quantum-visualizations/qmsolve/blob/main/CITATION.cff

**Comment by MarkDPhysics** [comment]
Thanks for the citation reference.  Here is the latest version of my
preprint:
https://zenodo.org/records/11456201
Deriving Heisenberg's operator algebra from a classical model of stochastic
mechanics
It's on the Zenodo preprint server which is run by CERN.  I prefer it over
arxiv.  The paper has passed peer review, and it is slated to appear as a
chapter in a book next year celebrating the 100th year anniversary of the
birth of quantum mechanics in 1925 with Heisenberg's paper.  I should be
able to incorporate your new citation reference.

Mark
